lib/glimesh_web/templates/oauth2_provider/application/form.html.eex

```diff
@@ -0,0 +1,27 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= label f, :name %>
+  <%= text_input f, :name %>
+  <%= error_tag f, :name %>
+
+  <br/>
+
+  <%= label f, :redirect_uri %>
+  <%= textarea f, :redirect_uri %>
+  <%= error_tag f, :redirect_uri %>
+  <span class="help-block">Use one line per URI</span>
+  <%= if is_nil(@native_redirect_uri) do %>
+    <span class="help-block">
+      Use <code><%= @native_redirect_uri %></code> for local tests
```

Comment:
This one is a little harder:
`<%= gettext("Use %{native_redirect_uri} for local tests", native_redirect_uri: @native_redirect_uri) %>`